.github/workflows/release.yml

```diff
@@ -97,8 +97,6 @@ jobs:
         GIT_EMAIL: ${{ secrets.VBR_GIT_EMAIL }}
         GIT_AUTHOR_EMAIL: ${{ secrets.VBR_GIT_EMAIL }}
         GIT_COMMITTER_EMAIL: ${{ secrets.VBR_GIT_EMAIL }}
```

Comment:
How do we prevent the integration tests are run here?

Reply:
no idea